demo/README.md

```diff
@@ -8,24 +8,49 @@ The following will provide a Tekton pipeline that can build an example python ap
 
 The details of each build type and the steps required to support Pelorus can be found in the [tekton manifest](./tekton-demo-setup/03-build-and-deploy.yaml)
 
-Example:
+
+Instructions:
+
+* Fork the konveyor/pelorus git repository to your github organization.
+
+* Add or configure your github-secret with your Github token
+
 ```
```

Comment:
I would not jump lines before opening a code block, because it get bigger lines then the unordered list items

Reply:
good point